src/views/organisations/Edit.vue

```diff
@@ -66,9 +68,11 @@ export default {
         name: this.organisation.name,
         slug: this.organisation.slug,
         description: this.organisation.description,
-        url: this.organisation.url,
-        email: this.organisation.email || '',
-        phone: this.organisation.phone || '',
+        url: this.organisation.url || "",
+        email: this.organisation.email || "",
+        phone: this.organisation.phone || "",
+        social_medias: this.organisation.social_medias,
+        location_id: this.organisation.location.id,
```

Comment:
Should be `location_id`.

```suggestion
        location_id: this.organisation.location_id,
```

Reply:
No this is correct. The location property of the returned organisation object is a full location object, not an ID reference.

Reply:
What would happen if we try to edit an organisation that doesn't already have an associated location? Would we not get a `trying to access property of null` error?

Reply:
Fixed. Ternary operator checks for existence of organisation.location.